docs/docs/migrate-from/google-dialogflow-to-rasa.mdx

```diff
@@ -15,13 +15,16 @@ Navigate to your agent's settings by clicking the gear icon.
 
 
 
-<img alt="Dialogflow Export" src={useBaseUrl("/img/dialogflow_export.png")} width="240" />
+<img caption="Selecting settings" src={useBaseUrl("/img/dialogflow_export.png")} width="240"
+alt="In DialogFlow, the gear icon that indicates settings sits next to the dropdown menu for
+choosing between projects."/>
 
 Click on the 'Export and Import' tab and click on the 'Export as ZIP' button.
 
 
 
-<img alt="Dialogflow Export 2" src={useBaseUrl("/img/dialogflow_export_2.png")} width="675" />
+<img caption="Selecting Export and Import" src={useBaseUrl("/img/dialogflow_export_2.png")} width="675"
+alt="The Settings page will let you select from multiple headers, of which you want to select Export and Import."/>
```

Comment:
Same comments here, re optimizing images with `Image` and versioning them locally